package.json

```diff
@@ -81,6 +81,11 @@
     "nanoid": "3.1.31",
     "glob-parent": "6.0.2",
     "simple-get": "4.0.1",
-    "semver": "7.5.3"
+    "semver": "7.5.3",
+    "homedir-polyfill": "../_EXCLUDED_",
```

Comment:
overrides section is mostly for transitive dependencies. There is more than one direct dependency that brings in homedir-polyfill. "npm list --depth=10" will display everything. Regarding Contrib file, since it has setup instructions, I don't think it is the right place to add this information. I have updated the jira attached to this PR with the reason why we are making this change.

Reply:
Please add the reasons to the commit comments themselves. Given that this is an open source project, but not all users have access to JIRA, we want these exclusions to be publicly documented in the codebase itself. And unless there's a way to do that in package.json, the commit comments are the best place to very clearly explain why each exclusion exists - and also why each dependency is being added here that wasn't added before (my assumption being that they need to be added since they were previously brought in via a now-excluded dependency).